src/pages/gateway/fastly.md

```diff
@@ -0,0 +1,297 @@
+
+  **NOTE**: The `Priority` of each VCL snippet determines the order in which VCL subroutines are executed. The following `Priority` fields only apply to the default configuration of Adobe Commerce. If you have other custom snippets, you will need to adjust the priorities accordingly.
+
+   - Allows API Mesh to function as a [Fastly backend](https://developer.fastly.com/reference/vcl/declarations/backend/):
+     - **Name** - api_mesh_backend
+     - **Type** - **init**
+     - **Priority** - **1**
+     - **Content**:
+  
+        ```csharp
+        backend F_graph_prod_adobe_io {
+            .always_use_host_header = true;
+            .between_bytes_timeout = 10s;
+            .connect_timeout = 1s;
+            .dynamic = true;
+            .first_byte_timeout = 15s;
+            .host = "graph.adobe.io";
+            .host_header = "graph.adobe.io";
+            .max_connections = 200;
+            .port = "443";
+            .share_key = "XXXXXXXXXXXXXXXX";
+            .ssl = true;
+            .ssl_cert_hostname = "graph.adobe.io";
+            .ssl_check_cert = always;
+            .ssl_sni_hostname = "graph.adobe.io";
+            .probe = {
+                .dummy = true;
+                .initial = 5;
+                .request = "HEAD / HTTP/1.1" "Host: graph.adobe.io" "Connection: close";
+                .threshold = 1;
+                .timeout = 2s;
+                .window = 5;
+            }
+        }
+        ```
+
+   - Enables the bypass header in API Mesh:
+     - **Name** - api_mesh_recv
+     - **Type** - **recv**
+     - **Priority** - **10**
+     - **Content**:
+
+        ```csharp
+        if (req.http.x - commerce - bypass - fastly - cache == "true") {
+          return (pass);
+        }
+        ```
+
+   - Determines what GraphQL can be cached:
+     - **Name** - api_mesh_recv2
+     - **Type** - **recv**
+     - **Priority** - **60**
+     - **Content**:
+
+        ```csharp
+        if ((req.request == "GET" || req.request == "HEAD") && (req.url.path~"/graphql" || req.url "^/api/(.*)") && req.url.qs~"query=") {
+          set req.http.graphql = "1";
+        }
+        else {
+          unset req.http.graphql;
+        }
+        if (req.url.path!~"/graphql" && req.url!~"^/api/(.*)") {
+          set req.http.Magento - Original - URL = req.url;
+
+          set req.url = querystring.regfilter(req.url, "^(utm_.*|gclid|gdftrk|_ga|mc_.*|trk_.*|dm_i|_ke|sc_.*|fbclid)$");
+        }
+        ```
+
+   - Cache miss - replace the `<mesh_id>` and `<mesh_api_key>` placeholders with the information from your API Mesh URL, which has the following structure: `https://graph.adobe.io/api/<meshId>/graphql?api_key=<your_apiKey>`. You can retrieve this information by running an [`aio api-mesh:describe`](./command-reference.md#aio-api-meshdescribe) command:
+     - **Name** - api_mesh_miss
+     - **Type** - **miss**
+     - **Priority** - **60**
```

Comment:
You have multiple snippets with a priority value of 60 and 10 (maybe others). Is that OK?

Reply:
I believe the priority value is specific to the Fastly operator. @amolina-adobe can you confirm?

Reply:
I think that's fine. The values provided are for a working configuration. For reference, all snippets added through the Adobe Commerce admin get a default priority of 50 unless otherwise specified. https://experienceleague.adobe.com/docs/commerce-cloud-service/user-guide/cdn/custom-vcl-snippets/fastly-vcl-custom-snippets.html?lang=en
![image](https://github.com/AdobeDocs/graphql-mesh-gateway/assets/44038475/5f2a503e-b5fd-4e90-b862-5ff9c064f4e8)